### DISCUSSION TITLE
Support .netConfig aliases

### DISCUSSION THREAD

**Discussion by danpowell88** [danielpalme/ReportGenerator#701] [Ideas]
It would be super handy to be able to specify for ReportGenerator to use a .netconfig alias particular for use cases when making reusable workflows

ie
```
[ReportGenerator]
	reports = coverage.xml

[ReportGenerator "test"]
	assemblyfilter = +Test
	assemblyfilter = -Test

[ReportGenerator "test2"]
	assemblyfilter = +Test2
	assemblyfilter = -Test2
```

Then use something like `-netConfigAlias:test2`

It would be handy if it loaded the non alias config and then overrode it with the aliased values, but equally you could just redefine the values in each alias if that is easier.

**Comment by danielpalme** [comment]
I will have look within the next days. Then I will decide if this makes sense.

Are you aware that you can use hierarchal configuration (current directory and any ancestor directories)?
You can specify common configuration in a parent directory and the individual options in nested configs.
Maybe this is useful for your scenario.

**Reply by danpowell88** [reply]
I wasn't aware of that I will see if it works as well
